observability/readme.md

```diff
@@ -2,6 +2,22 @@
 
 Building observable systems enables development teams at CSE to measure how well the application is behaving.
 
+## Table of Contents
```

Comment:
Should we start adding these TOCs more consistently?
In source code and code reviews we follow this pattern https://github.com/microsoft/code-with-engineering-playbook/tree/master/code-reviews - i.e. **Table of Contents** is bolded and not a title, and only one level is visible in the TOC - level 2 titles.

If you prefer this way, we should maybe start a conversation in the teams channel so that we can make all teams conform to whatever decision is made